apps/glusterfs/block_volume_entry_create.go

```diff
@@ -0,0 +1,96 @@
+//
+// Copyright (c) 2015 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package glusterfs
+
+import (
+	"github.com/boltdb/bolt"
+	"github.com/heketi/heketi/executors"
+	"github.com/lpabon/godbc"
+)
+
+func (v *BlockVolumeEntry) createBlockVolume(db *bolt.DB,
```

Comment:
Should add comment that this function requires that a volume be created and exist for it to create a block device.